cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -6,51 +6,66 @@
 import pytest
 
 import cfme.intelligence.chargeback.assignments as cb
-import cfme.intelligence.chargeback.rates as rates
 from cfme import test_requirements
 from cfme.infrastructure.provider import InfraProvider
 from cfme.infrastructure.provider.scvmm import SCVMMProvider
 from cfme.markers.env_markers.provider import ONE_PER_TYPE
 from cfme.services.dashboard import Dashboard
 from cfme.utils.appliance import ViaSSUI
-from cfme.utils.blockers import GH
 from cfme.utils.log import logger
 from cfme.utils.version import current_version
 from cfme.utils.wait import wait_for
 
 
 pytestmark = [
-    pytest.mark.meta(server_roles="+automate", blockers=[GH('ManageIQ/integration_tests:7479')]),
+    pytest.mark.meta(server_roles="+automate"),
     pytest.mark.usefixtures('uses_infra_providers'),
     test_requirements.ssui,
     pytest.mark.long_running,
     pytest.mark.ignore_stream("upstream"),
-    pytest.mark.provider([InfraProvider], selector=ONE_PER_TYPE,
-                         required_fields=[['provisioning', 'template'],
-                                          ['provisioning', 'host'],
-                                          ['provisioning', 'datastore']],
-                         scope="module"),
+    pytest.mark.provider(
+        [InfraProvider],
+        selector=ONE_PER_TYPE,
+        required_fields=[['provisioning', 'template'],
+                         ['provisioning', 'host'],
+                         ['provisioning', 'datastore']],
+        scope="module"
+    ),
 ]
 
 
 @pytest.fixture(scope="module")
-def new_compute_rate(enable_candu):
+def enable_candu(appliance):
+    candu = appliance.collections.candus
+    server_info = appliance.server.settings
+    original_roles = server_info.server_roles_db
+    server_info.enable_server_roles(
+        'ems_metrics_coordinator', 'ems_metrics_collector', 'ems_metrics_processor')
+    candu.enable_all()
+    yield
+    server_info.update_server_roles_db(original_roles)
+    candu.disable_all()
+
+
+@pytest.fixture(scope="module")
+def new_compute_rate(appliance, enable_candu):
```

Comment:
@mshriver , Why can't we use the global enable_candu fixture as is  without having to redefine a local fixture in this module ?

Reply:
The fixtures are modifying slightly different roles, so I'm not trying to change that in this PR. I'm trying to move forward the collections changes made by Landon originally.

I'd happily accept a PR from you changing this test to use global fixtures, if that's appropriate, following this merge. :)